src/Microsoft.Identity.Web.TokenAcquisition/DefaultTokenAcquirerFactoryImplementation.cs

```diff
@@ -112,10 +113,9 @@ private void CheckServiceProviderNotNull()
             }
         }
 
-
-        private static string GetKey(string? authority, string? clientId)
+        private static string GetKey(string? authority, string? clientId, string? region)
         {
-            return $"{authority}{clientId}";
+            return $"{authority}{clientId}{region}";
```

Comment:
what happens if the region is null?

Reply:
@sammurrayms can you check on this one and then we can merge. Thx.

Reply:
Added a test for this case. The resulting string just ignores the region.

```csharp
var no_region = DefaultTokenAcquirerFactoryImplementation.GetKey("1", "2", null);
Assert.Equal("12", no_region);
```